osu.Desktop/OsuGameDesktop.cs

```diff
@@ -206,5 +210,23 @@ protected override void Dispose(bool isDisposing)
             base.Dispose(isDisposing);
             osuSchemeLinkIPCChannel?.Dispose();
         }
+
+        private class SDL2BatteryInfo : BatteryInfo
```

Comment:
Class name not happy with inspections, either need to add an acronym to dotsettings ore rename.